src/order.rs

```diff
@@ -0,0 +1,247 @@
+//! The maker creates an order that defines how much he wants to buy for the amount he is selling.
```

Comment:
Would remove this comment - I wrote most of it at some point but (especially with changing the model) it is more confusing than in helps.